src/components/paymentForm/PaymentForm.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import '../../assets/main.css';
+import { useTranslation } from 'react-i18next';
+import { MDBContainer, MDBRow, MDBCol, MDBIcon, MDBInput } from 'mdbreact';
+
+export default function PaymentForm() {
+  const { t } = useTranslation();
+  return (
+    <MDBContainer>
+      <MDBRow>
+        <MDBCol md="4">
+          <form>
+            <p className="h5 mx-5 my-5 text-gray-800">
+              {t('PaymentPage.paymentMethod')}
```

Comment:
you can extract all the translation texts only once as object and then using them in your component

`const PaymentPageTexts = t(paymentPage, {returnObject: true})`